docs/guides/validation/how_to_validate_data_by_running_a_checkpoint.md

```diff
@@ -0,0 +1,147 @@
+---
+title: How to validate your data using a Checkpoint
+---
+
+import Prerequisites from '../../guides/connecting_to_your_data/components/prerequisites.jsx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide will help you validate your data by running a Checkpoint.
+
+As stated in [Validate your data using a Checkpoint](../../tutorials/getting_started/validate_your_data.md), the best way
+to validate data in production with Great Expectations is using a [Checkpoint](../../reference/checkpoints_and_actions).
+The advantages of a [Checkpoint](../../reference/checkpoints_and_actions) are ease of use, while combining the existing
+- [Expectation Suites](../../reference/expectations/expectations),
+- [DataConnectors](../../tutorials/getting_started/connect_to_data.md),
+- [BatchRequests](../../reference/datasources#Batches), and
+- [Validation Actions](../../reference/checkpoints_and_actions.md)
+in order to set up and perform the validation.  Otherwise, configuring these validation parameters would have to be done
```

Comment:
Just a nitpicky stylistic suggestion
```suggestion
The advantages of a [Checkpoint](../../reference/checkpoints_and_actions) are ease of use, by combining the existing configuration in order to set up and perform the validation:

- [Expectation Suites](../../reference/expectations/expectations)
- [DataConnectors](../../tutorials/getting_started/connect_to_data.md)
- [BatchRequests](../../reference/datasources#Batches)
- [Validation Actions](../../reference/checkpoints_and_actions.md)

Otherwise, configuring these validation parameters would have to be done
```

Reply:
@anthonyburdi This looks like the previous PR got merged, even though it was not supposed to yet.  I will have to address somehow.  Thanks.

Reply:
Fixed!